src/components/AudioPlayer/Buttons/AudioSpeedMenu.tsx

```diff
@@ -0,0 +1,36 @@
+import useTranslation from 'next-translate/useTranslation';
```

Comment:
1. use 14px equivalent. 
2. Add a right arrow
3. Use icon next to the active 
4. Add a divider under a playback speed
5. When you highlight over the title, only give a background hover to the back icon

Reply:
nit: AudioPlaybackSpeedMenu